### DISCUSSION TITLE
Is there a way to add Wappalyzer API key in HTTPX?

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/httpx#1229] [Q&A]
I have a Wappalyzer account and I want to discover more technologies while running HTTPX. Is it possible to add wappalyzer api key?

**Comment by MetzinAround** [comment]
Thanks for the question! I don't think there's a way to app the wappalyzer api key here, but I will double check for you.

**Comment by Mzack9999** [comment]
It will probably be possible with https://github.com/projectdiscovery/httpx/issues/959 either via chrome extension/js injection
